index.html

```diff
@@ -6,12 +6,62 @@
         <title>FB Model Configuration</title>
         <link rel="stylesheet" href="styles/style.css" />
     </head>
-    <body>
+    <body> 
         <header>
             <h1>Social Media Simulator</h1>
             <img src="images/logo.svg" alt="" class="logo" />
         </header>
         <main>
+        <!-- Dialog for Onboarding Step 1 -->
+        <dialog id="onboarding1" class="onboarding-modal">
+            <div class="onboarding-box">
+                <h2>Welcome to the Social Media Simulator</h2>
+                <p>This simulator allows users to configure the structure of social networks and news articles, and visualizes the spread of fake news. This project supports the research of Responsible IT at the Amsterdam University of Applied Sciences into the role of AI in the dissemination of misinformation.</p>
+                <button id="next1">Volgende</button>
+                <button id="skipOnboarding">Skip</button>
+            </div>
+        </dialog>
+
+        <!-- Dialog for Onboarding Step 2 -->
+        <dialog id="onboarding2" class="onboarding-modal">
+            <div class="onboarding-box">
+                <section>
```

Comment:
Section is niet nodig. Section is voor een deel van een pagina wat een heading heeft.

Reply:
fixed